Framework/PythonInterface/mantid/plots/helperfunctions.py

```diff
@@ -993,3 +995,20 @@ def line_colour_fill(ax):
             i = i + 1
 
     ax.get_figure().canvas.draw()
+
+
+# Update the color bar scale on a given image
+def update_colorbar_scale(figure, image, scale, vmin, vmax):
+    if vmin == 0:
```

Comment:
Can I just check. Should we not just do this adjustment for log scales?

Reply:
The check for log scale was removed in #27237 but I've just tested it with the check back in and it seems to work fine so I'll add it back in unless @PhilColebrooke can remember the reason it was removed.

Reply:
Thanks. It won't really hurt much but I think it's best if we only fiddle with the scale if absolutely necessary.

Reply:
I remember it being necessary but I don't remember why, probably just a side-effect of there being different ways to change the colorbar scale, each with their own code.